### PR TITLE
Gchat flow implementation

### PR DESCRIPTION
**Pull request by rudragoudapatilacc**
Gchat flow implementation

### CODE REVIEW

File: app/components/RequestUnlockModal.tsx [hunk 1]
```diff
@@ -11,8 +12,15 @@ import {
 
 import { useCheckOutData } from "@/app/hooks/useCheckOutData";
 const RequestUnlockModal = (props: any) => {
-    const { currentMetaData } = props;
+    const { currentMetaData, appSdk } = props;
     const { contextData } = useCheckOutData();
+    const GchatModel = async () => {
+        const userId = currentMetaData.updated_by;
+        const userEmail =appSdk.stack.getData().collaborators?.find((collaborator: any) => collaborator.uid === userId)?.email
+        console.log("userEmail :::::::::::: ",userEmail);
```
Comment (nicholas-russell-accenture) [inline app/components/RequestUnlockModal.tsx:20]: Please remove console.log

File: app/components/RequestUnlockModal.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
```
Comment (nicholas-russell-accenture) [inline app/components/RequestUnlockModal.tsx:1]: Please remove this override. Then, remove any unused dependencies for lint to succeed.

**Review by nicholas-russell-accenture — CHANGES_REQUESTED** [review]
Please see comments.

File: app/components/RequestUnlockModal.tsx (hunk 2 above)
Comment (rudragoudapatilacc) [reply app/components/RequestUnlockModal.tsx]: removed unwanted dependencies

File: app/components/RequestUnlockModal.tsx (hunk 1 above)
Comment (rudragoudapatilacc) [reply app/components/RequestUnlockModal.tsx]: removed unwanted console logs

**Comment by nicholas-russell-accenture** [comment]
Merging as requested fixes were applied and reviewed.

**Pull request merged**